ivy/functional/backends/paddle/experimental/statistical.py

```diff
@@ -99,6 +99,34 @@ def _validate_quantile(q):
     return True
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64", "float16")},
```

Comment:
float16 is not supported in Paddle. You should remove it! 😄

Reply:
> float16 is not supported in Paddle. You should remove it! 😄

This was not causing the error which I was getting but yes its true paddle dosent support float16. Passed all the test cases .
<img width="836" alt="image" src="https://github.com/unifyai/ivy/assets/82304628/4761007b-9814-4ab7-8d44-2305a60ef059">